src/components/DpSearchField/DpSearchField.stories.mdx

```diff
@@ -0,0 +1,28 @@
+import { Meta } from '@storybook/addon-docs'
+import DpSearchField from './DpSearchField'
+
+<Meta
+    title="Components/SearchField"
+    component={DpSearchField}
+/>
+
+# SearchField
+
+The DpSearchField component to add input field with attribute `type="search"` and a button to submit search processing.
+
+The DpSearchField component has two events `search & reset`.
```

Comment:
For events, i'd suggest to also not document them here but in the component code. Please [see the styleguidist docs](https://vue-styleguidist.github.io/docs/Documenting.html#events) for details.

Reply:
Done in https://github.com/demos-europe/demosplan-ui/pull/492/commits/969fb0c587ad25677a50fff83f41fc0f452b582e

Reply:
What do you think about also adding some basic event documentation in the component?

Reply:
Hm.. as i see, Storybook does not display the DocBlocks above event docblocks by default. So there is more research to be done here.